tests/bluechi_test/container.py

```diff
@@ -63,6 +63,15 @@ def gather_journal_logs(self, data_dir: str) -> None:
 
         self.get_file(log_file, data_dir)
 
+    def gather_coverage(self, data_dir: str) -> None:
+        gcno_file_location = "/usr/share/bluechi-coverage/"
+        gcda_file_location = "/root/rpmbuild/BUILD/bluechi-0.6.0/redhat-linux-build/src"
```

Comment:
Wouldn't gcda files be created in `/usr/libexec` for our daemons and in `/usr/bin` for `bluechictl`?